sdks/java/extensions/sorter/pom.xml

```diff
@@ -97,6 +97,10 @@
                   <pattern>org.apache.hadoop</pattern>
                   <shadedPattern>org.apache.beam.repackaged.org.apache.hadoop</shadedPattern>
                 </relocation>
+                <relocation>
+                  <pattern>com.google.common</pattern>
+                  <shadedPattern>org.apache.beam.repackaged.com.google.common</shadedPattern>
+                </relocation>
               </relocations>
             </configuration>
           </execution>
```

Comment:
Shading is tricky.. can you run `mvn package` and open the resulting jar to ensure that only `org.apache.beam` code is present?

(Doing this before fixing the Guava shading should show you a problem with com.google.thirdparty, I believe/)